### PR TITLE
[WebKit] Apple fixed issue #15593

### PR DESCRIPTION
**Pull request by SotoiGhost**

### CODE REVIEW

File: tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore [hunk 1]
```diff
@@ -2,7 +2,3 @@
 !missing-selector! WKWebView::findInteraction not bound
 !missing-selector! WKWebView::isFindInteractionEnabled not bound
 !missing-selector! WKWebView::setFindInteractionEnabled: not bound
-
-## https://github.com/xamarin/xamarin-macios/issues/15593
-!missing-selector! WKWebpagePreferences::isLockdownModeEnabled not bound
-!missing-selector! WKWebpagePreferences::setLockdownModeEnabled: not bound
```
Comment (rolfbjarne) [inline tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore:8]: Nitpick: missing end of line.

File: tests/xtro-sharpie/common-WebKit.ignore [hunk 2]
```diff
@@ -2,7 +2,3 @@
 !missing-selector! WKWebView::findInteraction not bound
 !missing-selector! WKWebView::isFindInteractionEnabled not bound
 !missing-selector! WKWebView::setFindInteractionEnabled: not bound
-
-## https://github.com/xamarin/xamarin-macios/issues/15593
-!missing-selector! WKWebpagePreferences::isLockdownModeEnabled not bound
-!missing-selector! WKWebpagePreferences::setLockdownModeEnabled: not bound
```
Comment (rolfbjarne) [inline tests/xtro-sharpie/common-WebKit.ignore:8]: Same

**Review by rolfbjarne — APPROVED** [review]

**Review by chamons — APPROVED** [review]

File: tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore (hunk 1 above)
Comment (SotoiGhost) [reply tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore:8]: I think the missing EOL symbol is from the before changes. The new changes includes an EOL.

File: tests/xtro-sharpie/common-WebKit.ignore (hunk 2 above)
Comment (SotoiGhost) [reply tests/xtro-sharpie/common-WebKit.ignore:8]: https://github.com/xamarin/xamarin-macios/pull/15717#discussion_r951635325

File: tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore (hunk 1 above)
Comment (rolfbjarne) [reply tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore:8]: -EMONDAYMORNING 🤦

**Review by dalexsoto — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Comment by dalexsoto** [comment]
Crashed Known issue: https://github.com/xamarin/maccore/issues/581

**Pull request merged**